src/actors/generic.py

```diff
@@ -0,0 +1,134 @@
+import logging
+from abc import ABC, abstractmethod
+
+from thespian.actors import ActorAddress, ActorTypeDispatcher
+
+from src.utils.address_book import AddressBook
+from src.utils.constants import ActorNames, Requests
+
+
+class GenericActor(ActorTypeDispatcher, ABC):
```

Comment:
Some of the stuff here isnt very generic, the looping part especially is very specififc to the ultrasonic sensor. I dont think it needs to be defined in the "Generic" class, we can do it in ultrasonic.